src/tron/TronEngine.ts

```diff
@@ -1575,7 +1587,7 @@ export class TronEngine extends CurrencyEngine<TronTools, SafeTronWalletInfo> {
     const { edgeSpendInfo, currencyCode } = super.makeSpendCheck(
       edgeSpendInfoIn
     )
-    const { memos = [] } = edgeSpendInfo
+    const { memos = [], tokenId } = edgeSpendInfo
```

Comment:
We should probably refactor `makeSpendCheck` to return a `tokenId` as well.

This can also be a follow-up task.

Reply:
https://app.asana.com/0/1200382638405084/1206891509258298/f